web/themes/custom/novel/templates/fields/field--node--field-subtitle--list-teaser.html.twig

```diff
@@ -1,5 +1,3 @@
-<p>
-  {% for item in items %}
-    {{ item.content }}
-  {% endfor %}
-</p>
+{% for item in items %}
+  {{ item.content }}
+{% endfor %}
```

Comment:
I'm unsure if this template is used/necessary, or is a leftover from [this commit](https://github.com/danskernesdigitalebibliotek/dpl-cms/commit/c0434c73bf5c03752b0bbe453b80e1cf4b9d415d#diff-cf874922e4745a51aecacb001512f6a8f38febaec430dd708d1955dc1151d53d)

Reply:
It is needed because is else will use `web/themes/custom/novel/templates/fields/field--field-subtitle.html.twig`